src/include/common.h

```diff
@@ -18,17 +18,18 @@ extern uint8_t UID[6];
 extern uint8_t MasterUID[6];
 extern uint16_t CRCInitializer;
 
-typedef enum
+typedef enum : uint8_t
 {
-    TLM_RATIO_NO_TLM = 0,
-    TLM_RATIO_1_128 = 1,
-    TLM_RATIO_1_64 = 2,
-    TLM_RATIO_1_32 = 3,
-    TLM_RATIO_1_16 = 4,
-    TLM_RATIO_1_8 = 5,
-    TLM_RATIO_1_4 = 6,
-    TLM_RATIO_1_2 = 7
-
+    TLM_RATIO_STD = 0,   // Use suggested ratio from ModParams
```

Comment:
should this come after TLM_RATIO_NO_TLM (which ends up as default upon first flash)

Reply:
was wrong about `Off` becoming default on first flash. I have 1:128 set before flashing and it carried over to the PR flashing. 1:128 was index 1, and on this PR, that becomes Off.

Reply:
I disagree. The first option is the default, Std, then come all the manual rates (Off is a manual rate), then Race (expert rate). Race is as far as possible from Std so someone doesn't switch from Std and say "Race? Oh that must be better performance!" without ever looking at the docs and seeing it is strictly for Racing without TLM.